### DISCUSSION TITLE
blog/00-micro-changelog

### DISCUSSION THREAD

**Discussion by giscus** [trevortylerlee/astro-micro#10] [Announcements]
# blog/00-micro-changelog

Features, enhancements, and changes.

https://astro-micro.vercel.app/blog/00-micro-changelog

**Comment by tsavpyn** [comment]
Hell ya, looks great!

**Reply by EdvardSolomon** [reply]
# A Hell ya!

**Comment by Zain-ul-din** [comment]
amazing!

**Comment by mayura-andrew** [comment]
nice

**Comment by LarryEmerson12** [comment]
Incredible!

**Comment by Kushal2004** [comment]
Good one mate

**Comment by LarryEmerson12** [comment]
looks way too good

**Comment by JFelixZuniga** [comment]
amazing!

**Comment by without1022** [comment]
11

**Comment by galangthejokrstudio** [comment]
Mantap

**Comment by plworawit** [comment]
Great!!!

**Comment by InTEGr8or** [comment]
I really like the Giscuss

**Comment by InTEGr8or** [comment]
How is this installed? It would be nice if there was a command or link for new Astro-ers.

**Reply by trevortylerlee** [reply]
[Set up instructions for Giscus](https://astro-micro.vercel.app/blog/01-getting-started#set-up-giscus)

**Comment by ans-inayat** [comment]
sdf

**Comment by jeff-wolff** [comment]
cool

**Reply by trevortylerlee** [reply]
:)

**Comment by gauthamk** [comment]
wow!

**Comment by code-gritt** [comment]
Nice blog!

**Comment by dongqing66** [comment]
it's beautiful
